### ISSUE TITLE
[BUG] FatalThrowableError after installing the prequel

### ISSUE THREAD

**Issue by luigel** [Protoqol/Prequel#30]
**Describe the bug**
After installing the prequel package using composer, I can't make run of the `php artisan route:list` command. 

**Screenshots**
![image](https://user-images.githubusercontent.com/26235960/60933512-1c7aae00-a2f5-11e9-8456-60b05939b6cf.png)

**Comment by QuintenJustus** [comment]
Hm, that's a serious error. Could you post the results of `php artisan route:list -v`?

**Comment by xuanquynh** [comment]
Which `protoqol/prequel` version did you installed? I think the version 0.6.2-beta has already fixed it.

**Comment by luigel** [comment]
> Hm, that's a serious error. Could you post the results of `php artisan route:list -v`?

@QuintenJustus I copied and paste the result

```
 php artisan route:list -v

   Symfony\Component\Debug\Exception\FatalThrowableError  : Argument 1 passed to Doctrine\Common\Inflector\Inflector::singularize() must be of the type string, null given, called in D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Support\Pluralizer.php on line 86

  at D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\doctrine\inflector\lib\Doctrine\Common\Inflector\Inflector.php:441
    437|      * @param string $word The word in plural form.
    438|      *
    439|      * @return string The word in singular form.
    440|      */
  > 441|     public static function singularize(string $word) : string
    442|     {
    443|         if (isset(self::$cache['singularize'][$word])) {
    444|             return self::$cache['singularize'][$word];
    445|         }

  Exception trace:

  1   Doctrine\Common\Inflector\Inflector::singularize()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Support\Pluralizer.php:86

  2   Illuminate\Support\Pluralizer::singular()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Support\Str.php:445

  3   Illuminate\Support\Str::singular()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\protoqol\prequel\src\Classes\Database\DatabaseTraverser.php:103

  4   Protoqol\Prequel\Classes\Database\DatabaseTraverser::getModel()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\protoqol\prequel\src\Http\Requests\PrequelDatabaseRequest.php:61

  5   Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest::getValidatorInstance()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Validation\ValidatesWhenResolvedTrait.php:23

  6   Illuminate\Foundation\Http\FormRequest::validateResolved()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Providers\FormRequestServiceProvider.php:30

  7   Illuminate\Foundation\Providers\FormRequestServiceProvider::Illuminate\Foundation\Providers\{closure}(Object(Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest), Object(Illuminate\Foundation\Application))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:1082

  8   Illuminate\Container\Container::fireCallbackArray(Object(Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:1046

  9   Illuminate\Container\Container::fireAfterResolvingCallbacks("Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest", Object(Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:1031

  10  Illuminate\Container\Container::fireResolvingCallbacks("Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest", Object(Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:687

  11  Illuminate\Container\Container::resolve("Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest", [])
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:615

  12  Illuminate\Container\Container::make("Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest", [])
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Application.php:767

  13  Illuminate\Foundation\Application::make("Protoqol\Prequel\Http\Requests\PrequelDatabaseRequest")
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:925

  14  Illuminate\Container\Container::resolveClass(Object(ReflectionParameter))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:853

  15  Illuminate\Container\Container::resolveDependencies()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:818

  16  Illuminate\Container\Container::build("Protoqol\Prequel\Http\Controllers\DatabaseController")
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:667

  17  Illuminate\Container\Container::resolve("Protoqol\Prequel\Http\Controllers\DatabaseController", [])
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:615

  18  Illuminate\Container\Container::make("Protoqol\Prequel\Http\Controllers\DatabaseController", [])
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Application.php:767

  19  Illuminate\Foundation\Application::make("Protoqol\Prequel\Http\Controllers\DatabaseController")
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Routing\Route.php:233

  20  Illuminate\Routing\Route::getController()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Routing\Route.php:833

  21  Illuminate\Routing\Route::controllerMiddleware()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Routing\Route.php:794

  22  Illuminate\Routing\Route::gatherMiddleware()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Console\RouteListCommand.php:167

  23  Illuminate\Foundation\Console\RouteListCommand::getMiddleware(Object(Illuminate\Routing\Route))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Console\RouteListCommand.php:117

  24  Illuminate\Foundation\Console\RouteListCommand::getRouteInformation(Object(Illuminate\Routing\Route))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Console\RouteListCommand.php:89

  25  Illuminate\Foundation\Console\RouteListCommand::Illuminate\Foundation\Console\{closure}(Object(Illuminate\Routing\Route))
      [internal]:0

  26  array_map(Object(Closure))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Support\Collection.php:1120

  27  Illuminate\Support\Collection::map(Object(Closure))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Console\RouteListCommand.php:90

  28  Illuminate\Foundation\Console\RouteListCommand::getRoutes()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Console\RouteListCommand.php:74

  29  Illuminate\Foundation\Console\RouteListCommand::handle()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:32

  30  call_user_func_array([])
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:32

  31  Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:90

  32  Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Object(Closure))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:34

  33  Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), [])
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Container\Container.php:576

  34  Illuminate\Container\Container::call()
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Console\Command.php:183

  35  Illuminate\Console\Command::execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\symfony\console\Command\Command.php:255

  36  Symfony\Component\Console\Command\Command::run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Console\Command.php:170

  37  Illuminate\Console\Command::run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\symfony\console\Application.php:921

  38  Symfony\Component\Console\Application::doRunCommand(Object(Illuminate\Foundation\Console\RouteListCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\symfony\console\Application.php:273

  39  Symfony\Component\Console\Application::doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\symfony\console\Application.php:149

  40  Symfony\Component\Console\Application::run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Console\Application.php:90

  41  Illuminate\Console\Application::run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php:133

  42  Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
      D:\WORK EVOTECH\INVENTORY-MANAGEMENT\inventory-management\artisan:37
```

**Comment by luigel** [comment]
> Which `protoqol/prequel` version did you installed? I think the version 0.6.2-beta has already fixed it.

@xuanquynh I installed 0.6.2-beta version. Still having the error.

**Comment by QuintenJustus** [comment]
Ah! I think I see what’s going wrong, will get back to you!

**Comment by xuanquynh** [comment]
I see. I will take a look.

**Comment by xuanquynh** [comment]
I discover the error and will submit a PR soon.

**Comment by xuanquynh** [comment]
This will be fixed https://github.com/Protoqol/Prequel/pull/31

**Comment by QuintenJustus** [comment]
Confirmed fix in #31 closing now.
